force-app/main/default/classes/NPSP_Address.cls

```diff
@@ -408,11 +408,7 @@ public inherited sharing class NPSP_Address implements IAddress {
         if (String.isBlank(countryCode)) {
             return;
         }
-
-        if (String.isEmpty(address.MailingCountry__c)) {
-            address.MailingCountry__c = orgConfig.validCountriesByCode().get(countryCode
-                    .toUpperCase());
-        }
+        address.MailingCountry__c = orgConfig.validCountriesByCode().get(countryCode.toUpperCase());
```

Comment:
This fix works great, but as usual I'm worried about any possible regressions. @daniel-fuller @force2b Can either of you think of scenarios this is breaking? It's basically forcing the Country to match the Country code, which I think is a good thing, but I can't help but think there was a reason for the isEmpty check!

Reply:
Agreed on these points